packages/stylex/src/StyleXCSSTypes.js

```diff
@@ -1243,6 +1244,8 @@ export type CSSProperties = $ReadOnly<{
   initialLetterAlign?: all | initialLetterAlign,
   inlineSize?: all | inlineSize,
 
+  interpolateSize: all | interpolateSize,
```

Comment:
You're missing a question mark here, which is causing type errors. Please fix it so I can merge it. Thanks!

```suggestion
  interpolateSize?: all | interpolateSize,
```